live-examples/html-examples/forms/button.html

```diff
@@ -0,0 +1,9 @@
+<p><button>Default button</button></p>
+
+<p><button disabled>Disabled button</button></p>
+
+<p><button name="submit" type="submit" value="submit-true">
+  Form submit button
+</button></p>
```

Comment:
I would put this all on one line as the others are but, might make this one a little hard to read as it will auto-wrap. So perhaps:

```
<p>
  <button name="submit" type="submit" value="submit-true">
    Form submit button
  </button>
</p>
```